go/vt/mysqlctl/backup.go

```diff
@@ -125,6 +126,20 @@ func Backup(ctx context.Context, params BackupParams) error {
 		return vterrors.Wrap(err, "unable to get backup storage")
 	}
 	defer bs.Close()
+
+	params.Logger.Infof("Checking if we should parameterize backup storage %s?", backupstorage.BackupStorageImplementation)
+	if p, ok := bs.(backupstorage.Parameterizer); ok {
```

Comment:
I didn't have to do this. I could have added new params or methods to the `BackupStorage` or `FileHandle` interfaces. But I wasn't sure how careful we need to be about making changes to those APIs, so figured best to do things in a non-breaking way until I get some guidance.